Migrating deployment docs (#154)
CodeWithAloha/HIERR: src/docs/deployment.md

# HIERR Application Assumptions
1. Windows Server 2022

> Some additional components may need to be installed if this is not the version of Windows used

Reviewer: ```suggestion
> Some additional components may need to be installed if this is not the version of Windows used
```

Reviewer: https://github.com/lifeparticle/Markdown-Cheatsheet

> # HIERR Application Download link
> You will be prompted to download this file in the instructions below.
> 
> [Releases · CodeWithAloha/HIERR · GitHub](https://github.com/CodeWithAloha/HIERR/releases)

Reviewer: We can probably just remove this section and hyperlink the releases page below

```suggestion
```

# Update Instructions

> 1. Download the HIERR application package from the link above.

Reviewer: ```suggestion
1. Download the HIERR application package from the [releases page](https://github.com/CodeWithAloha/HIERR/releases).
```
